### PR TITLE
Cordoned servers are preferred first when decreasing allocation count

### PR DESCRIPTION
**Pull request by jackwaudby**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1127.surge.sh

File: modules/ROOT/pages/clustering/databases.adoc [hunk 1]
```diff
@@ -72,6 +72,8 @@ Keep in mind that during such a transition, the database will be unavailable for
 `ALTER DATABASE` commands are optionally idempotent, with the default behavior to fail with an error if the database does not exist.
 Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.
 
+If an `ALTER DATABASE` command would decrease the number of allocations of a database, allocations on xref:clustering/servers.adoc#_cordoned_servers[cordoned servers] are removed first.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/databases.adoc:75]:
```suggestion
If the `ALTER DATABASE` command decreases the number of allocations of a database, allocations on xref:clustering/servers.adoc#_cordoned_servers[cordoned servers] are removed first. 
```

File: modules/ROOT/pages/clustering/servers.adoc [hunk 2]
```diff
@@ -88,6 +88,7 @@ Additionally, servers which are deallocating will not have any further databases
 
 The _Cordoned_ state is similar to _Deallocating_ in that servers in this state will not be allocated to host additional databases.
 Unlike _Deallocating_ however, cordoned servers do not lose the databases they already host.
+It is worth noting when decreasing the number of allocations of a database, allocations on cordoned servers are removed first.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/clustering/servers.adoc:91]:
```suggestion
It is worth noting that when decreasing the number of allocations of a database, allocations on cordoned servers are removed first. 
```

**Review by AlexicaWright — COMMENTED** [review]
Just some nit-picking grammar. Thanks @jackwaudby !

**Comment by jackwaudby** [comment]
Thanks @AlexicaWright 🚀

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**